gradlew

```diff
@@ -115,7 +114,7 @@ case "$( uname )" in                #(
   NONSTOP* )        nonstop=true ;;
 esac
 
-CLASSPATH=$APP_HOME/gradle/wrapper/gradle-wrapper.jar
+CLASSPATH="\\\"\\\""
```

Comment:
This is from the gradle distribution? It looks odd...

Reply:
Yes, it seems to be an empty string, added when this command is run: `sh gradlew wrapper --gradle-version 8.14.2`. Earlier the gradle-wrapper.jar was used in classpath, but now it is used with -jar option. so CLASSPATH is kept empty.